crowdsec-docs/docs/intro.mdx

```diff
@@ -38,12 +47,39 @@ In addition to the core "detect and react" mechanism, CrowdSec is committed to s
 ## Architecture
 
 
+<!-- @kka : update schema:
+    - align text
+    - rename data sources to log sources
+    - move log sources out of the SE box
+-->
 <div style={{display: 'flex'}}>
     <div style={{textAlign: 'center', flex: '1'}}>
-    <img src={useBaseUrl('/img/crowdsec_architecture.png')}></img>
+    <img src={useBaseUrl('/img/simplified_SE_underthehood.png')}></img>
     </div>
 </div>
 
+Under the hood, the Security Engine has various components:
+ 
+ - The Log Processor is in charge of detection: it analyzes logs from [various data sources](/docs/data_sources/intro) or [HTTP requests](/appsec/intro.md) from web servers.
+ - The [WAF](/appsec/intro.md) feature is part of the Log Processor and filters HTTP Requests from the compatible web servers. 
```

Comment:
Are we calling this small part of the SE the WAF?  
I was still on the fact that we can call an install CrowdSec WAF when talking in general but in this sentence case it would be the **AppSec capability/processor** or something like that:

- An [AppSec processor](/appsec/intro.md) is part of the Log Processor and filters HTTP Requests from the compatible web servers and RPs.